packages/process/src/node/process.ts

```diff
@@ -41,22 +46,16 @@ export abstract class Process {
         return this._killed;
     }
 
-    set killed(killed: boolean) {
-        /* readonly public property */
-    }
-
```

Comment:
can this be ommited ?
what happens on process.killed = boolean ?

Reply:
> process.killed = boolean

That is a compiler error. Having a property getter is equivalent to a `readonly` property from the client code's point of view. Let me know if you meant something else.

Reply:
OK good :) that's what I wanted